src/components/sbb-timetable-row/sbb-timetable-row.tsx

```diff
@@ -1,75 +1,227 @@
+    return items;
+  }
+
+  private _clickHandler = (): void => {
+    this.sbbClick.emit({
+      bubbles: true,
+    });
+  };
 
   public render(): JSX.Element {
-    const config = JSON.parse(this.config);
+    if (this.loadingTrip) {
+      return this._renderSkeleton();
+    }
+
+    const { legs, notices, situations, id }: InterfaceTimetableRowAttributes['trip'] =
+      this.trip || {};
 
-    this._getAdditionalStyleClasses(config);
+    const {
+      product,
+      direction,
+      departureWalk,
+      departure,
+      arrival,
+      arrivalWalk,
+      tripStatus,
+      occupancy,
+      duration,
+    } = this.trip?.summary || {};
+    const sortedNotices = this._sortPriority(notices);
+    const sortedSituations = this._sortPriority(situations);
 
     return (
-      <div class={`timetable__row ${this._additionalTimetableRowClasses.join(' ')}`} role="none">
-        <sbb-timetable-row-header
-          config={JSON.stringify(config.rowHeader)}
-          role="rowheader"
-        ></sbb-timetable-row-header>
-        <sbb-timetable-transportation-details
-          config={JSON.stringify(config.details)}
-          role="gridcell"
-        ></sbb-timetable-transportation-details>
-        <sbb-timetable-row-button expanded={false} role="gridcell"></sbb-timetable-row-button>
-        <sbb-timetable-platform
-          config={JSON.stringify(config.platform)}
-          role="gridcell"
-        ></sbb-timetable-platform>
-        <sbb-timetable-occupancy
-          config={JSON.stringify(config.occupancy)}
-          role="gridcell"
-        ></sbb-timetable-occupancy>
-        <sbb-timetable-travel-hints
-          config={JSON.stringify(config.travelHints)}
-          role="gridcell"
-        ></sbb-timetable-travel-hints>
-        {/* <sbb-timetable-park-and-rail
-          config={JSON.stringify(config.parkAndRail)}
-          role='gridcell'
-        >
-        </sbb-timetable-park-and-rail> */}
-        <sbb-timetable-duration
-          config={JSON.stringify(config.duration)}
-          role="gridcell"
-        ></sbb-timetable-duration>
-        <sbb-timetable-cus-him
-          config={JSON.stringify(config.cusHim)}
-          role="gridcell"
-        ></sbb-timetable-cus-him>
-      </div>
+      <sbb-card
+        active={this.acitve}
+        id-value={id}
+        accessibility-label={this.accessibilityLabel}
+        onClick={this._clickHandler}
+      >
+        {this.loadingPrice && <span slot="badge" class="sbb-loading__badge"></span>}
+        {this.price && !this.loadingPrice && (
+          <sbb-card-badge
+            slot="badge"
+            appearance={this.price.isDiscount ? 'primary' : 'primary-negative'}
+            price={this.price.price}
+            text={this.price.text}
+            isDiscount={this.price.isDiscount}
+          />
+        )}
+        <div class="sbb-timetable__row">
+          <div class="sbb-timetable__row-header">
+            <div class="sbb-timetable__row-details">
```

Comment:
I think product was not really read by screen reader (voice over, mac). (E.g IR 27 I never heard)

Reply:
I would also prefer to implement this one later.

Reply:
ok for me :)